src/app/chat/views/message_format.nim

```diff
@@ -19,7 +19,7 @@ proc renderInline(self: ChatMessageList, elem: TextItem): string =
   of "emph": result = fmt("<span style=\"font-style: italic;\">{elem.literal}</span> ")
   of "strong": result = fmt("<span style=\"font-weight: bold;\">{elem.literal}</span> ")
   of "link": result = elem.destination
-  of "mention": result = fmt("<span style=\"color: #000000;\">{self.mention(elem.literal)}</span> ")
+  of "mention": result = fmt("<a href=\"//{elem.literal}\" style=\"font-weight: bold; color: #000000;\">@{self.mention(elem.literal)}</a> ")
```

Comment:
We want mentions to be clickable so they have to be anchor tags. We also need to attach the pubkey of the mention to that element so we can retrieve it from QML (unfortunately, the raw pk information is diluted via `message_format` along the way otherwise). Anyways, one way to attach that pubkey is to put it in the `href`.

I first thought I'd go with something like `href="@0xpk"` or `href="status-profile://0xpk"` but both variants get filtered out by our XSS protection, rendering empty hrefs.

We could use `href="#0xpk` but technically, these will already be channels (although we could obviously detect if the `#` is followed by a `0x...`, but still, it'd be a special (and probably unexpected) behaviour.

That's why I went with a `//` prefix for. It survives the XSS filter and is unique enough for us to make a distinction. 

If you have any other ideas let me know!